### ISSUE TITLE
Different Accuracy with different batch-size

### ISSUE THREAD

**Issue by abhiagwl4262** [bearpaw/pytorch-pose#101]
I am getting significant difference in the validation accuracy when I am using different batch size. Can anyone please explain this ?

**Comment by bearpaw** [comment]
Please check https://github.com/bearpaw/pytorch-pose/issues/53
